python/tvm/runtime/vm.py

```diff
@@ -552,6 +561,11 @@ def benchmark(
             milliseconds. This can be used to ensure that the function is run enough to get an
             accurate measurement.
 
+        end_to_end : bool
+            If set, include time to transfer input tensors to the device and time to transfer
+            returned tensors in the total runtime. This will give accurate timings for end to end
```

Comment:
i'm not sure "accurate" is exactly the right thing to say here since there is probably jitter in the transfer...but the gist makes sense.